proto/sentry_protos/snuba/v1/endpoint_get_traces.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+message TraceColumn {
+  enum Name {
+    NAME_UNSPECIFIED = 0;
+    NAME_TRACE_ID = 1;
+    NAME_START_TIMESTAMP = 2;
+    NAME_ROOT_SPAN_NAME = 3;
+    NAME_TOTAL_SPAN_COUNT = 4;
+    NAME_FILTERED_SPAN_COUNT = 5;
```

Comment:
what is filtered span count?

Reply:
`FILTERED_SPAN_COUNT` is the number of spans respecting the conditions where `TOTAL_SPAN_COUNT` is always the count of spans in a trace, regardless of conditions.